src/System.Threading.Overlapped/ref/System.Threading.Overlapped.cs

```diff
@@ -41,4 +41,31 @@ public sealed partial class ThreadPoolBoundHandle : System.IDisposable
         [System.CLSCompliantAttribute(false)]
         public unsafe static object GetNativeOverlappedState(System.Threading.NativeOverlapped* overlapped) { throw null; }
     }
+    public partial class Overlapped
+    {
+        public Overlapped() { }
+        [System.ObsoleteAttribute("This constructor is not 64-bit compatible.  Use the constructor that takes an IntPtr for the event handle.  http://go.microsoft.com/fwlink/?linkid=14202")]
+        public Overlapped(int offsetLo, int offsetHi, int hEvent, System.IAsyncResult ar) { }
+        public Overlapped(int offsetLo, int offsetHi, System.IntPtr hEvent, System.IAsyncResult ar) { }
+        public System.IAsyncResult AsyncResult { get { throw null; } set { } }
+        [System.ObsoleteAttribute("This property is not 64-bit compatible.  Use EventHandleIntPtr instead.  http://go.microsoft.com/fwlink/?linkid=14202")]
+        public int EventHandle { get { throw null; } set { } }
+        public System.IntPtr EventHandleIntPtr { get { throw null; } set { } }
+        public int OffsetHigh { get { throw null; } set { } }
+        public int OffsetLow { get { throw null; } set { } }
+        [System.CLSCompliantAttribute(false)]
```

Comment:
CLSComplaint attributes should remain in the refs as the compilers to look at them.